docs/metrics.md

```diff
@@ -2,9 +2,20 @@
 
 This is a list of metrics exported by Knative Eventing components.
 
-## Broker ingress
+## Broker
 
-| Name             | Type      | Description                  | Tags               |
-| ---------------- | --------- | ---------------------------- | ------------------ |
-| `messages_total` | count     | Number of messages received. | `result`, `broker` |
-| `dispatch_time`  | histogram | Time to dispatch a message.  | `result`, `broker` |
+These are exported by `broker-ingress` pods.
+
+| Name                    | Type      | Description                  | Tags               |
+| ----------------------- | --------- | ---------------------------- | ------------------ |
+| `broker_events_total`   | count     | Number of events received.   | `result`, `broker` |
+| `broker_dispatch_time`  | histogram | Time to dispatch an event.   | `result`, `broker` |
+
+## Trigger
+
+These are exported by `broker-filter` pods.
+
+| Name                   | Type      | Description                  | Tags                                           |
+| ---------------------- | --------- | ---------------------------- | -----------------------------------------------|
+| `trigger_events_total` | count     | Number of events received.   | `result`, `broker`, `trigger`                  |
+| `trigger_filter_time`  | histogram | Time to filter an event.     | `result`, `broker`, `trigger`, `filter_result` |
```

Comment:
Missing the `error` tag.

Reply:
As you point out later, the `error` tag is unused so I'll just remove it from the View definition.